lineapy/plugins/kubeflow_pipeline_writer.py

```diff
@@ -175,9 +175,10 @@ def get_rendered_task_definitions(
                 loading_blocks=loading_blocks,
                 pre_call_block="",
                 call_block=task_def.call_block,
-                post_call_block=task_def.post_call_block,
+                post_call_block="",
```

Comment:
didnt you need this to be lazar's version?

Reply:
yeah i just realized this and i fixed it

Reply:
some type of merge conflict I think